build.gradle.kts

```diff
@@ -165,6 +170,10 @@ tasks {
     withType<JavaCompile>().all {
         targetCompatibility = project.properties["jvmTarget"]?.toString() ?: "21"
         sourceCompatibility = project.properties["jvmTarget"]?.toString() ?: "21"
```

Comment:
Not a thing with this PR specifically, but maybe rather than setting `-target` and `source`, you want to configure the `--release`?
```kotlin
    options.compilerArgs.add("--release")
    options.compilerArgs.add(project.properties["jvmTarget"]?.toString() ?: "21")
```
Why? [Because](https://stackoverflow.com/q/43102787/1207769).